pkg/storage/gcs/client.go

```diff
@@ -22,16 +21,9 @@ var (
 
 // NewGcsClient reads the gcp secrets in the operator's namespace and uses
 // them to create a new client for accessing the GCS API.
-func NewGcsClient(kubeClient client.Client) (*gstorage.Client, error) {
-	var err error
-
-	namespace, err := k8sutil.GetOperatorNamespace()
-	if err != nil {
-		return nil, fmt.Errorf("failed to get operator namespace: %v", err)
-	}
-
+func NewGcsClient(kubeClient client.Client, namespace string) (*gstorage.Client, error) {
 	secret := &corev1.Secret{}
-	err = kubeClient.Get(context.TODO(),
+	err := kubeClient.Get(context.TODO(),
```

Comment:
Why are we doing this, instead of defining `var err error` as above?

Reply:
After removing `namespace, err := k8sutil.GetOperatorNamespace()`, the code block was left as:
```
var err error

if err != nil {
        return nil, fmt.Errorf("failed to get operator namespace: %v", err)
}
```
When I which seemed really silly, so I removed that in and moved the declaration into the first assignment.

I have no strong conviction about the definition not being a separate line and can add it back in if you'd prefer it that way.